packages/host/tests/acceptance/interact-submode-test.gts

```diff
@@ -1501,6 +1501,68 @@ module('Acceptance | interact submode tests', function (hooks) {
       assert.dom('[data-test-workspace-chooser]').exists();
     });
 
+    test<TestContextWithSave>('can create a card when 2 stacks are present', async function (assert) {
```

Comment:
this test was one of many ways to reveal the card creation outside of a CardResource. previously this would just barf with an exception.